hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmVolumeArgs.java

```diff
@@ -177,10 +176,14 @@ public OmOzoneAclMap getAclMap() {
     return aclMap;
   }
 
-  public LongAdder getUsedBytes() {
+  public long getUsedBytes() {
     return usedBytes;
   }
 
+  public void setUsedBytes(long usedBytes) {
+    this.usedBytes = this.usedBytes + usedBytes;
```

Comment:
Method name setUsedBytes seems confused, can we rename to incrUsedBytes(long bytes)